docs/es/docs/contributing.md

```diff
@@ -0,0 +1,497 @@
+
+Aqui están los pasos para ayudar con las traducciones.
+
+#### Tips y guías
+
+* Verifica los <a href="https://github.com/fastapi/fastapi/pulls" class="external-link" target="_blank">pull requests existentes</a> para tu idioma. Puedes filtar los pull requests por la label de tu idioma. Por ejemplo, para Español, la label es <a href="https://github.com/fastapi/fastapi/pulls?q=is%3Aopen+sort%3Aupdated-desc+label%3Alang-es+label%3Aawaiting-review" class="external-link" target="_blank">`lang-es`</a>.
+
+* Revisa esos pull requests, solicitando cambios o aprobandolos. Para los lenguajes que no hablo, esperaré a que los otros revisen las traducciones antes de realizar el merge.
+
+/// tip
+
+Puedes <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request" class="external-link" target="_blank">agregar comentarios con sugerencias de cambio</a> en un pull request existente.
+
+Consulta la documentación acerca de <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews" class="external-link" target="_blank"> cómo agregar una revisión de pull request</a> para aprobarlo o pedir cambios.
+
+///
+
+* Verifica en <a href="https://github.com/fastapi/fastapi/discussions/categories/translations" class="external-link" target="_blank">GitHub Discussion</a> para coordinar traducciones en tu idioma. Puedes suscribirte a él, y cuando haya un nuevo pull request para revisar, un comentario automático será añadido a la discución.
+
+* Si traduces páginas, agregar un único pull request por cada página traducida. Esto hará mucho mas fácil de revisar por otros.
+
+* Consulta los códigos de 2-letras para el lenguaje que quieres traducir, puedes usar la tabla<a href="https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes" class="external-link" target="_blank">Lista de códigos ISO 639-1</a>.
+
+#### Idioma existente
+
+Digamos que quieres traducir una página para un idioma que ya tiene traducciones para algunas páginas, como el Español.
+
+En el caso del Español, el código de 2-letras sería `es`. Así que, el directorio para las traducciones en Español están ubicadas en `docs/es/`.
+
+/// tip
+
+El idioma principal ("oficial") es el inglés, y se encuentra ubicado en `docs/en/`.
+
+///
+
+Ahora ejecuta el live serrver para los documentos en español:
+
+<div class="termy">
+
+```console
+// Usa el comando "live" y pásale el código de idioma como argumento en la CLI
+$ python ./scripts/docs.py live es
+
+<span style="color: green;">[INFO]</span> Serving on http://127.0.0.1:8008
+<span style="color: green;">[INFO]</span> Start watching changes
+<span style="color: green;">[INFO]</span> Start detecting changes
+```
+
+</div>
+
+/// tip
+
+De otra manera, puedes hacer los pasos de los scripts manualmente.
+
+Ve al directorio del idioma, para las traducciones al Español sería en `docs/es/`:
```

Comment:
```suggestion
Ves al directorio del idioma, para las traducciones al Español sería en `docs/es/`:
```

Reply:
What do you mean? `Ves` is for the see verb in Spanish, `Ve` is for the go verb, In the English version it's using the Go verb:

```
Go into the language directory, for the main docs in English it's at `docs/en/`:
```

Reply:
My fault, "ves" is a common error in Spanish to say "go" as non-formal. As a first idea, "ve" sounds like a formal use for "usted", so I changed it to "ves". But "ve" is the correct use for "tú" or "vos"